lib/src/test/java/uk/gov/di/ipv/cri/passport/library/service/ClientAuthJwtIdServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.di.ipv.cri.passport.library.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.cri.passport.library.persistence.DataStore;
+import uk.gov.di.ipv.cri.passport.library.persistence.item.ClientAuthJwtIdItem;
+
+import java.time.Instant;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ClientAuthJwtIdServiceTest {
+    @Mock private DataStore<ClientAuthJwtIdItem> mockDataStore;
+
+    @Mock private ConfigurationService mockConfigurationService;
+
+    private ClientAuthJwtIdService clientAuthJwtIdService;
```

Comment:
```suggestion
    @InjectMocks private ClientAuthJwtIdService clientAuthJwtIdService;
```
Then you can delete the setUp method

Reply:
Didn't know you could do this, nice! Done.